packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -51,6 +51,39 @@ String mainAxisAssetImageUrl(MainAxisAlignment alignment) {
   return 'assets/img/story_of_layout/main_axis_alignment/${describeEnum(alignment)}.png';
 }
 
+List<double> computeRenderSizes({
+  @required Iterable<double> sizes,
+  @required double smallestSize,
+  @required double largestSize,
+  @required double smallestRenderSize,
+  @required double largestRenderSize,
+  @required double maxSize,
+  bool shouldForceToOccupyMaxSize = true,
+}) {
+  final transformToRenderSize = (largestRenderSize) => [
+        for (var s in sizes)
+          (s - smallestSize) *
+                  (largestRenderSize - smallestRenderSize) /
+                  (largestSize - smallestSize) +
+              smallestRenderSize
+      ];
+  var renderSizes = transformToRenderSize(largestRenderSize);
+  if (shouldForceToOccupyMaxSize) {
```

Comment:
This is the implementation of the formula that we discussed earlier.
Thanks @DaveShuckerow !! :-)